CorsixTH/Src/th_lua_gfx.cpp

```diff
@@ -43,6 +43,35 @@ SOFTWARE.
 #include FT_TYPES_H
 
 namespace {
+uint32_t flags_from_table(lua_State* L, int index) {
```

Comment:
Are you aware that the draw flags also exist as named constants in Lua?

```
Lua/utility.lua
217:DrawFlags = {}
218:DrawFlags.FlipHorizontal  = 2^0
219:DrawFlags.FlipVertical    = 2^1
220:DrawFlags.Alpha50         = 2^2
221:DrawFlags.Alpha75         = 2^3
222:DrawFlags.AltPalette      = 2^4
223:DrawFlags.EarlyList       = 2^10
224:DrawFlags.ListBottom      = 2^11
225:DrawFlags.BoundBoxHitTest = 2^12
226:DrawFlags.Crop            = 2^13
```

`utility` also has a number of bit operation functions (`flag_*` functions around lines 190-210, and two more around lines 295-310)